docs/user_guide.md

```diff
@@ -0,0 +1,37 @@
+# Using S2I images
+
+## Invoking scripts embedded into an image
+
+When there is a requirement for **adding custom logic** to [S2I scripts](builder_image.md#s2i-scripts), it is possible to execute standartd scripts inside of custom scripts from user’s repository. Note that it only works when you have to add logic (for example, to generate some static assets after executing `assemble` script) and not for the case of modifying this logic. In latter case you have to replace standard scripts.
```

Comment:
I did updates for other comments, but I'm stuck with this: we have 2 variants already but none of them we can use and I cannot come up with another one :(